mkt/constants/regions.py

```diff
@@ -108,12 +108,22 @@ class RESTOFWORLD(REGION):
     if country.get('ratingsbody'):
         country['ratingsbody'] = getattr(ratingsbodies, country['ratingsbody'])
 
+    # This if check can be removed once marketplace-constants is updated.
```

Comment:
Its returning mcc as a list in the next version. So once 0.5.0 is out, it will always be a list and we remove this.